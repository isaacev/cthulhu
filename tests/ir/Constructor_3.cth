type Foo =
  | Bar(Str);

#[entry]
fn main() -> () {
  Bar { x: 123 };
}
