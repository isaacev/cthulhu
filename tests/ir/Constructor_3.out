
  UNKNOWN CONSTRUCTOR FIELD  -------------------------------------------------

  Cannot construct an instance of:

    ::Constructor_3::Foo::Bar

  Use of an unknown field:

    x

    : Constructor_3
  6 |   Foo::Bar { x: 123 };
    |              ^

